Testing/SystemTests/tests/analysis/GUIStartupTest.py

```diff
@@ -58,13 +56,14 @@ def runTest(self):
         # good startup
         p = subprocess.Popen(self.cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
         out, err = p.communicate()
-        self.assertEquals(out, b'Hello Mantid\n')
+        # Do not check line ending as it's different on Windows
+        self.assertTrue(b'Hello Mantid' in out)
```

Comment:
This assertion fails for me - `out` is empty and `err` contains: `'python: can\\'t open file \\'\\scripts\\ErrorReporter\\error_dialog_app.py\\': [Errno 2] No such file or directory'`